src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
 
+class Header {
+  get logo() {
+    return f;
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the ' + `${f}`;
+  };
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return y;
+  };
+  clickOnCommentButton() {
+    return 'Click on the ' + y;
```

Comment:
should contain clickOnCommentButton method. The method should return a string 'Click on the' + commentButton. `y` is not a comment button property